src/pages/home/index.jsx

```diff
@@ -1,14 +1,20 @@
 import { useProductsQuery } from '../../api/products/getProducts.js';
 import ProductsList from '../../components/products-list/index.jsx';
 import { Container } from './styles.js';
-
+import SpinnerLoading from '../../components/SpinnerLoading';
 const Home = () => {
-  const { data } = useProductsQuery();
+  const { arrayOfProducts, isLoading, error } = useProductsQuery();
 
   return (
-    <Container>
-      <ProductsList data={data} />
-    </Container>
+    <>
+      {isLoading || error ? (
```

Comment:
let add constant called it shouldDisplaySpinner => const shouldDisplaySpinner=isLoading || error;
.
.
<>shouldDisplaySpinner? ( . . .